packages/core-js/internals/object-create.js

```diff
@@ -26,13 +37,29 @@ var NullProtoObject = function () {
   iframe.src = String(JS);
   iframeDocument = iframe.contentWindow.document;
   iframeDocument.open();
-  iframeDocument.write(LT + SCRIPT + GT + 'document.F=Object' + LT + '/' + SCRIPT + GT);
+  iframeDocument.write(scriptTag('document.F=Object'));
   iframeDocument.close();
-  NullProtoObject = iframeDocument.F;
-  while (length--) delete NullProtoObject[PROTOTYPE][enumBugKeys[length]];
-  return NullProtoObject();
+  return iframeDocument.F;
 };
 
+// Check for document.domain and active x support
+// No need to use active x approach when document.domain is not set
+// see https://github.com/es-shims/es5-shim/issues/150
+// variation of https://github.com/kitcambridge/es5-shim/commit/4f738ac066346
+var activeXDocument;
+try {
+  /* global ActiveXObject */
+  activeXDocument = document.domain && new ActiveXObject('htmlfile');
```

Comment:
Here should be used delayed initialization - we should not create `ActiveXObject` or `IFrame` if `Object.create` is available.

Reply:
If I understood correctly, it's a reason why currently tests fails - you try to create an `IFrame` on Node.

Reply:
Ooops...

Reply:
I just avoid initialization when `Object.create` is available